sbnanaobj/StandardRecord/SREXTInfo.cxx

```diff
@@ -0,0 +1,15 @@
+#include "sbnanaobj/StandardRecord/SREXTInfo.h"
+
+#include <limits>
+#include <climits>
+
+namespace caf
+{
+  SREXTInfo::SREXTInfo():
+    gates_since_last_trigger(std::numeric_limits<unsigned int>::max()),
+    isBNBOffBeam(false),
+    isNuMIOffBeam(false),
+    isMajority(false),
+    isMinBias(false)
```

Comment:
These should be commented what they mean or their name should be changed to something more generic.